File: Jour2/jour2.c
#include<stdio.h>

int main()
{
	float a,b;
	printf(" ========== Ce programme vous permet d'effectuer la somme de deux nombres. ==========\n");
	
	printf("Donner le premier nombre : ");
	scanf("%f",&a);
	// Attention aucun control n'est effectuer

	printf("Donner le second nombre : ");
	scanf("%f",&b);
	// Attention aucun control n'est effectuer

	printf("La somme de %.2f et de %.2f vaut : %.2f \n",a,b,a+b);
	return 0;
}

File: Jour2/jour2.pas
program Somme;
var
a,b : real;
begin
	writeln(' ========== Ce programme vous permet de faire la somme de deux nombres. ==========');
	write('Donner le premier nombre : ');
	read(a);
	write('Donner le second nombre : ');
	read(b);
	writeln('La somme de ',a:0:2, ' et ', b:0:2, ' est : ', (a +b):0:2);
end.